.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules/
 /coverage
 package-lock.json
 /dist/
+.idea/
```

Comment:
Can you please remove it? Cos this is an IDE specific ignore.

There is a nice post of creating a global ignore
https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer